WMIAdventure/frontend/src/js/components/global/atoms/CompactCardView/CompactCardView.js

```diff
@@ -58,7 +58,7 @@ class CompactCardView extends React.Component {
     setStateFromProps = () => {
         this.props.cardName ? this.setState({cardName: this.props.cardName}) : null;
         this.props.cardLevel ? this.setState({cardLevel: this.props.cardLevel}) : null;
-        this.props.cardImage ? this.setState({cardImage: this.props.cardImage}) : null;
+        this.setState({cardImage: this.props.cardImage});

```

Comment:
zawsze image będzie istnieć?

Reply:
czasem jest image w propsach jako `null`, ale niżej w kodzie jest to handlowane żeby się placeholder wyświetlał jak jest `null`